components/phpunit_bridge.rst

```diff
@@ -232,6 +232,48 @@ test::
 
 And that's all!
 
+.. caution::
+
+    The ``@group time-sensitive`` annotation is equivalent to ``ClockMock::register(MyTest::class)``,
+    so if you want to get a time based function mocked into one of the source code you will need to
```

Comment:
time-based (missing dash)
I don't understand "into one of the source code". Maybe "into another class"?